engine/src/srvcgi.cpp

```diff
@@ -707,12 +707,11 @@ static void cgi_store_form_urlencoded(MCVariable *p_variable, MCDataRef p_data,
 
 static void cgi_fix_path_variables()
 {
-	char *t_path, *t_path_end;
+    char *t_path;
 
 	MCStringRef env;
     env = nil;
-	t_path = nil;
-    t_path_end = nil;
+    t_path = nil;
     
     // SN-2014-07-29: [[ Bug 12865 ]] When a LiveCode CGI script has a .cgi extension and has
```

Comment:
I'm not clear here what you mean - the same processing seems to be being applied to t_path regardless of whether it came from PATH_TRANSLATED or SCRIPT_FILENAME.

Looking at this - http://stackoverflow.com/questions/2261951/what-exactly-is-path-info-in-php - it suggests that if PATH_INFO is not being set then its a problem with web-server config not the thing processing the CGI request.